modules/apps/object/object-js-components-web/src/main/resources/META-INF/resources/components/BaseEntryFields/BooleanEntryBaseField.tsx

```diff
@@ -0,0 +1,46 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import React from 'react';
+
+import {SingleSelect} from '../Select/SingleSelect';
+
+interface BooleanEntryBaseFieldProps {
+	error?: string;
+	label: string;
+	onChange: (selected: string | undefined) => void;
+	placeholder?: string;
+	required?: boolean;
+	selectedBooleanItem?: string | undefined;
```

Comment:
i believe here you can remove the undefined and only leave the string, since you made the changes to use an empty string